dotCMS/src/main/webapp/html/js/dotcms/dijit/RemotePublisherDialog.js

```diff
@@ -249,13 +248,32 @@ dojo.declare("dotcms.dijit.RemotePublisherDialog", null, {
         if (this.workflow && this.workflow.actionId) {
             this.container.evaluateCondition(this.workflow.actionId, this.title, eventData);
         } else {
-            var customEvent = document.createEvent("CustomEvent");
+            const customEvent = document.createEvent("CustomEvent");
             customEvent.initCustomEvent("ng-event", false, false,  {
                 name: "push-publish",
                 data: eventData
             });
             document.dispatchEvent(customEvent);
         }
+    },
+
+    _dispatchAngularWorkflowEvent: function (workflow) {
+        const data = {
```

Comment:
Codacy found an issue: [Avoid trailing commas in object or array literals](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190817)